### ISSUE TITLE
Unable to merge latest spec repo changes

### ISSUE THREAD

**Issue by sbc100** [WebAssembly/memory64#26]
When trying to run `update-tests.sh` in https://github.com/WebAssembly/testsuite the memory64 tests could not be updated to do unresolved conflicts merging `spec/master`.

```
>> git reset origin/main --hard
HEAD is now at 5bfb70d9 Added SIMD/threads memory ops to Memory64 proposal (#25)
>> git merge -q spec/master -m merged
Auto-merging README.md
CONFLICT (content): Merge conflict in README.md
Auto-merging document/core/index.rst
CONFLICT (content): Merge conflict in document/core/index.rst
Auto-merging document/core/util/macros.def
CONFLICT (content): Merge conflict in document/core/util/macros.def
Auto-merging interpreter/binary/decode.ml
CONFLICT (content): Merge conflict in interpreter/binary/decode.ml
Auto-merging interpreter/binary/encode.ml
CONFLICT (content): Merge conflict in interpreter/binary/encode.ml
Auto-merging interpreter/exec/eval.ml
CONFLICT (content): Merge conflict in interpreter/exec/eval.ml
Auto-merging interpreter/host/spectest.ml
CONFLICT (content): Merge conflict in interpreter/host/spectest.ml
Auto-merging interpreter/runtime/memory.ml
CONFLICT (content): Merge conflict in interpreter/runtime/memory.ml
Auto-merging interpreter/runtime/memory.mli
CONFLICT (content): Merge conflict in interpreter/runtime/memory.mli
Auto-merging interpreter/syntax/types.ml
CONFLICT (content): Merge conflict in interpreter/syntax/types.ml
Auto-merging interpreter/text/arrange.ml
CONFLICT (content): Merge conflict in interpreter/text/arrange.ml
Auto-merging interpreter/text/parser.mly
CONFLICT (content): Merge conflict in interpreter/text/parser.mly
Auto-merging interpreter/valid/valid.ml
CONFLICT (content): Merge conflict in interpreter/valid/valid.ml
Auto-merging test/core/binary-leb128.wast
Auto-merging test/core/binary.wast
CONFLICT (content): Merge conflict in test/core/binary.wast
Automatic merge failed; fix conflicts and then commit the result.
>> git checkout --ours document interpreter test/harness
Updated 12 paths from the index
>> git add document interpreter test/harness
>> git -c core.editor=true merge --continue
U	README.md
U	test/core/binary.wast
error: Committing is not possible because you have unmerged files.
```

**Comment by sbc100** [comment]
I think the only conflict that the testsuite cares about is `test/core/binary.wast`

**Comment by sbc100** [comment]
This was fixed in 48e69f39
